### PR TITLE
added requestkey attribute

### PR DESCRIPTION
**Pull request by triashu**
### Description

[Describe what this change achieves]

Adds "requestkey" attribute (if defined) in ntp.conf

### Issues Resolved
"requestkey" attribute is needed for ptp

[List any existing issues this PR resolves]

### Check List

- [ ] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] All commits have been signed for the Developer Certificate of Origin. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD>

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 license 'Apache-2.0'
 description 'Installs and configures ntp as a client or server'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version '3.3.1'
+version '3.3.2'
```
Comment (shortdudey123) [inline metadata.rb]: Chef bumps the version when they release, so this should be reverted

Comment (triashu) [inline metadata.rb]: @shortdudey123 Done.

**Comment by tas50** [comment]
Thanks

**Comment by triashu** [comment]
Thanks for merging. When does this show up in supermarket?

**Comment by tas50** [comment]
It looks like my previous push failed. I just pushed it up again.

**Comment by triashu** [comment]
Thanks @tas50

**Pull request merged**